hom/cogs/support.py

```diff
@@ -76,13 +76,15 @@ async def clear_ratelimit(self, user_id: int) -> None:
     async def mod_check(self, interaction: discord.Interaction[commands.Bot]) -> bool:
         assert isinstance(interaction.user, discord.Member)
 
-        if not (is_mod := utils.contains_roles(interaction.user.roles, "Moderator")):
+        if not any(r.id == Config.MOD_ROLE for r in interaction.user.roles):
```

Comment:
Might be a good idea to update Config.MOD_ROLE to Config.MOD_ROLE_ID since it's not really a role.

Reply:
Everything pulled from the environment is an ID, except the token. I don't really think it's a big deal.